9723 Re-add Pagination Styling

zachrog opened the pull request.

Commit:
9723: add usa-pagination styling

codyseibert committed:
devex-1180 - pulling in rosies work on split tests

Commit:
9723: alphabetize imports

codyseibert committed:
devex-1180 - trying to split unit tests and coverage

TomElliottFlexion reviewed (commented):
Should we backmerge this from staging to keep the branches synced up? Probably not a big deal either way

codyseibert committed:
1180 - trying to prevent tests from failing

codyseibert committed:
fixing artifcats

codyseibert committed:
devex-1180 - increase coverage

codyseibert committed:
devex-1180 - reducing number of runners

codyseibert committed:
devex-1180 - final clean up

codyseibert committed:
devex-1180 reduce runners move

codyseibert committed:
removing a comment

rosiehsmith committed:
devex-split-shared: splitting shared tests into 4 parallel on CI

rosiehsmith committed:
devex-split-shared: add coverage merging

devex-split-shared: rename artifact path

devex-shared: increase nodes so I can see results faster

devex-shared: update jest config, temporarily run more nodes to see results faster

devex-shared: renamed coverage artifacts

devex-shared: add tsx and jsx to moduleFileExtensions

devex-1180 removing unused lines

Revert "devex-split-shared: rename artifact path"

This reverts commit 1c8b0aecfba8c93e26c0f277fb8117f0543cbba0.

Revert "devex-shared: increase nodes so I can see results faster"

This reverts commit 63795d5d52a2740e2d720a5c9b7dd4accb474b7a.

codyseibert committed:
devex-1180 removing unused lines

codyseibert committed:
Merge pull request #10048 from flexion/devex-split-shared-tests

devex-split-shared: splitting shared tests into 4 parallel on CI

mmarcotte committed:
Merge pull request #3588 from flexion/devex-1180-slow-unit-tests

devex-1180 - pulling in rosies work on split tests [skip ci]

zachrog committed:
Merge branch 'staging' into 9723-usa-pagination-styling
